Client/Frontend/Strings.swift

```diff
@@ -5224,35 +5265,6 @@ extension String {
         comment: "Label for Cancel button")
 }
 
-// MARK: - SearchSettings
-extension String {
-    public static let SearchSettingsTitle = MZLocalizedString(
```

Comment:
We can't remove strings that are still used in previous version, until that string is entirely released in the last version it was used in. This is an ask by the l10n team. Please see `How to remove a string` in [this wiki](https://github.com/mozilla-mobile/firefox-ios/wiki/How-to-add-and-modify-Strings#how-to-remove-a-string).